v22.2/install-cockroachdb-linux.md

```diff
@@ -113,10 +111,12 @@ true
 </div>
 
 <div id="use-docker-linux" class="install-option">
-  <h3>Use Docker</h3>
+  <h2 id="install-docker">Use Docker</h2>
 
   {{site.data.alerts.callout_danger}}Running a stateful application like CockroachDB in Docker is more complex and error-prone than most uses of Docker. Unless you are very experienced with Docker, we recommend starting with a different installation and deployment method.{{site.data.alerts.end}}
 
+  <p>For CockroachDB v22.2.beta-5 and above, Docker images are <a href="https://docs.docker.com/build/building/multi-platform/">multi-platform images</a> that contains binaries for both Intel and ARM. CockroachDB on ARM systems is <b>experimental</b> and is not yet qualified for production use. Multi-platform images do not take up additional space on your Docker host.</p><p>Docker images for previous releases contain Intel binaries only. Intel binaries can run on ARM systems, but with a significant reduction in performance.</p>
```

Comment:
Nit: I personally use "earlier/later" to indicate preceding/subsequent versions as I think this is clearer. But I don't feel strongly about this.

Reply:
I personally think "later" is clear because I think of versions in terms of their ascending or descending numbers, not release dates. The reason I brought this up is because "above" could also imply "earlier" to me. But yeah, I'll just file a Style issue and we can resolve later.